bundles/org.openhab.binding.tellstick/src/main/java/org/openhab/binding/tellstick/internal/live/TelldusLiveDeviceController.java

```diff
@@ -276,37 +285,66 @@ public void onRequest(TellstickDeviceEvent newDevices) {
     }
 
     <T> T callRestMethod(String uri, Class<T> response) throws TelldusLiveException {
+        LocalDateTime start = LocalDateTime.now();
         T resultObj = null;
         try {
-            for (int i = 0; i < MAX_RETRIES; i++) {
-                try {
-                    resultObj = innerCallRest(uri, response);
-                    break;
-                } catch (TimeoutException e) {
-                    logger.warn("TimeoutException error in get", e);
-                } catch (InterruptedException e) {
-                    logger.warn("InterruptedException error in get", e);
-                }
-            }
-        } catch (JAXBException e) {
-            logger.warn("Encoding error in get", e);
+            resultObj = innerCallRest(uri, response);
+        } catch (TimeoutException | InterruptedException | ExecutionException | JAXBException | XMLStreamException e) {
```

Comment:
Not needed for `InterruptedException`:
```java
Thread.currentThread().interrupt();
```
?

Reply:
Probably better yes.
Done